go.mod

```diff
@@ -21,6 +23,7 @@ require (
 	github.com/stretchr/testify v1.8.0
 	github.com/vishvananda/netlink v1.2.1-beta.2.0.20220630165224-c591ada0fb2b
 	go.uber.org/goleak v1.1.12
+	go.uber.org/zap v1.19.1
```

Comment:
There is a new release 1.23.0 https://github.com/uber-go/zap/releases

Reply:
I have just done `go mod tidy`. How goes go select the versions?

Reply:
you can use `go mod edit -require go.uber.org/zap@v1.23.0` (or change manually in the go.mod) then `go mod tidy`, but otherwise I have no idea how go selects the versions.

Reply:
I edited go.mod